src/app/auth/google/google.component.html

```diff
@@ -31,9 +31,15 @@
       </div>
     </div>
     <div class="d-flex pt-15px justify-content-end">
-      <button (click)="finishFlow(true)" class="btn btn-primary font-weight-bold fs-15px sign-up-btn">
+      <button (click)="finishFlow(true)" class="btn btn-primary font-weight-bold fs-15px sign-up-btn" *ngIf="!globalVars.showJumio()">
         Continue to {{ globalVars.hostname }}
       </button>
+      <button (click)="startJumio()" class="btn btn-primary font-weight-bold fs-15px sign-up-btn" *ngIf="globalVars.showJumio()">
+        Get  <b>{{ globalVars.nanosToUSD(globalVars.jumioBitCloutNanos, 0)}} Free</b>
+      </button>
     </div>
   </div>
+  <div class="container home-container" *ngIf="showJumio">
```

Comment:
It's quite confusing to me that we have showJumio and `globalVars.showJumio()`. Can we give these variables different names?

Reply:
cool - this variable is now `showGetFreeCLOUT`